tools/bigquery-to-xml/bq_to_xml.py

```diff
@@ -0,0 +1,105 @@
+"""
```

Comment:
You need a #!/usr/bin/env python3 line here no ?

Reply:
done